nengo/utils/least_squares_solvers.py

```diff
@@ -253,7 +270,11 @@ class RandomizedSVD(LeastSquaresSolver):
     ``sklearn.utils.extmath.randomized_svd`` for details about the parameters.
     """
 
+    n_components = IntParam('n_components', low=1)
+    n_oversamples = IntParam('n_oversamples', low=0)
+
```

Comment:
Should add a `DictParam` for the kwargs too? In which case, it should maybe be passed in as an explicit dict, rather than as `**kwargs`?